tests/CloudFront/SignerTest.php

```diff
@@ -23,22 +23,36 @@ public function setUp()
         );
     }
 
+    /**
+     * Assert that the key is parsed during construction
+     *
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessageRegExp /PK .*Not a real private key/
+     */
+    public function testBadPrivateKeyContents() {
+        $privateKey = "Not a real private key";
+        $s = new Signer(
+            "not a real keypair id",
+            $privateKey
+        );
+    }
+
     /**
      * Assert that the key is parsed during construction
      *
```

Comment:
Should this doc string description be differentiated from the previous test (i.e. variable vs file)?